### PR TITLE
docs(async): use `Array.fromAsync()` in `debounce()` example

### LINKED ISSUE #4282: suggestion(async): Update `debounce` JSDoc example with `Array.fromAsync`
There are current JSDoc comments.

https://github.com/denoland/deno_std/blob/52138c6f99fb3cf59d690722a07eb140c2203acc/async/debounce.ts#L27-L37

BTW, `Array.fromAsync` implemented [Deno 1.38.0 ](https://deno.com/blog/v1.38#v8-120).

So, it could be able to replace without type annotation like below. 

```typescript
Array.fromAsync(
  Deno.watchFs('./'),
  debounce((event) => {
    console.log('[%s] %s', event.kind, event.paths[0]);
  }, 200),
);
```

### PR DESCRIPTION
**Pull request by Hajime-san**
resolves #4282

### CODE REVIEW

File: async/debounce.ts [hunk 1]
```diff
@@ -26,15 +26,12 @@ export interface DebouncedFunction<T extends Array<unknown>> {
  * ```
  * import { debounce } from "https://deno.land/std@$STD_VERSION/async/debounce.ts";
  *
- * const log = debounce(
- *   (event: Deno.FsEvent) =>
- *     console.log("[%s] %s", event.kind, event.paths[0]),
- *   200,
+ * Array.fromAsync(
```
Comment (iuioiua) [inline async/debounce.ts:29]:
```suggestion
 * await Array.fromAsync(
```

**Review by iuioiua — APPROVED** [review]
LGTM. Thank you!

**Comment by kt3k** [comment]
I feel this is rather confusing as we don't use the result of `Array.fromAsync()`

**Comment by iuioiua** [comment]
Perhaps. Though, we don't assign the result to a variable... Happy to revert if you prefer.

**Pull request merged**